frontend/src/Components/WorkstationCardMinimal.tsx

```diff
@@ -1,40 +1,80 @@
-import { Box, Center, Heading, useColorModeValue } from "@chakra-ui/react";
+import {
+  Alert,
+  AlertIcon,
+  Box,
+  Center,
+  HStack,
+  Heading,
+  Tooltip,
+  useColorModeValue,
+} from "@chakra-ui/react";
 import { WorkStationData } from "../Data";
+import { TimeIcon } from "@chakra-ui/icons";
+import { cropString } from "../Utils/Utils";
+
+const LAST_SEEN_WARN_THRESH = 60 * 5;
+const GREEN = "#25D36B";
 
 export const WorkstationCardMin = ({
   width,
-  data: { name, gpus },
+  data: { name, gpus, lastSeen },
 }: {
   width: number;
-  data: WorkStationData;
+  data: WorkStationData & { lastSeen: number | undefined };
 }) => {
   const textCol = useColorModeValue("black", "white");
   return (
     <Center>
       <Box
-        padding={2}
+        padding={0}
         w={width}
         rounded={"md"}
         bg={useColorModeValue("white", "gray.900")}
       >
-        <Heading size="lg" color={textCol}>
-          {name}
-        </Heading>
-        {gpus.map((s, i) => {
-          return (
-            <Box key={i}>
-              <Heading size="md">{`${s.gpu_name} (${(
-                s.memory_total / 1000
-              ).toFixed(0)} GB)`}</Heading>
-              <p>{`${s.gpu_util < 25 ? "🟢 Free" : "🔴 In-use"} (${Math.round(
-                s.gpu_util,
-              )}% Utilisation)`}</p>
-              <p>{`${s.gpu_temp < 80 ? "❄️" : s.gpu_temp < 95 ? "🌡️" : "🔥"} ${Math.round(
-                s.gpu_temp,
-              )} °C (${Math.round(s.fan_speed)}% Fan Speed)`}</p>
-            </Box>
-          );
-        })}
+        {lastSeen !== undefined && lastSeen > LAST_SEEN_WARN_THRESH ? (
+          <Alert
+            roundedTopLeft="md"
+            roundedTopRight="md"
+            textAlign="center"
+            status="warning"
+          >
+            <AlertIcon></AlertIcon>
+            {lastSeen === undefined
+              ? "Missing provenance data!"
```

Comment:
nit: I think we should look at what the best words are to use here (maybe just take from what they have now?)